config/test.exs

```diff
@@ -20,6 +20,15 @@ config :tesla, Lightning.UsageTracking.GithubClient, adapter: Tesla.Mock
 # to provide built-in test partitioning in CI environment.
 # Run `mix help test` for more information.
 #
+
+# If we are running a performance test, set the ownership timeout to infinity.
```

Comment:
Given the limited usage of this setting I opted to keep it in here rather than polluting `bootstrap.ex`.